app/models/user.rb

```diff
@@ -0,0 +1,4 @@
+class User < ActiveRecord::Base
+  has_many :spots, :through => :spot_students
+  has_many :spot_students

```

Comment:
I don't think we need this line?

Reply:
I pretty much follow this guide here. https://medium.com/@pk60905/many-to-many-relationship-in-rails-cf86e12db8b0.
Atm I don't know if SQL queries will correctly behave as we expect since we haven't made up fake data and played with the database. But the last time I did this for many-to-many relationship, so I think It can work. If later it doesnt work, we can change it though.